src/flypad-backend/src/ThirdParty/ThirdParty.cpp

```diff
@@ -0,0 +1,117 @@
+#include "ThirdParty.h"
+
+#define SELCAL_LIGHT_TIME_MS 300
+
+ThirdParty::ThirdParty(HANDLE hSimConnect): _hSimConnect(hSimConnect) {}
+
+void ThirdParty::initialize() {
+    _selcal = register_named_variable("A32NX_ACP_SELCAL");
+    _selcalReset = register_named_variable("A32NX_ACP_RESET");
+    _volumeCOM1ACP1 = register_named_variable("A32NX_ACP1_Volume_VHF1");
+    _volumeCOM1ACP2 = register_named_variable("A32NX_ACP2_Volume_VHF1");
+    _volumeCOM1ACP3 = register_named_variable("A32NX_ACP3_Volume_VHF1");
+    _volumeCOM2ACP1 = register_named_variable("A32NX_ACP1_Volume_VHF2");
+    _volumeCOM2ACP2 = register_named_variable("A32NX_ACP2_Volume_VHF2");
+    _volumeCOM2ACP3 = register_named_variable("A32NX_ACP3_Volume_VHF2");
+    _updateReceiversFromThirdParty = register_named_variable("A32NX_COM_UpdateReceiversFromThirdParty");
+
+    _isInitialized = true;
+
+    std::cout << "FLYPAD_BACKEND (ThirdParty): ThirdParty initialized" << std::endl;
+}
+
+void ThirdParty::shutdown() {
+    _isInitialized = false;
+    std::cout << "FLYPAD_BACKEND (ThirdParty): ThirdParty shutdown" << std::endl;
+}
+
+void ThirdParty::onUpdate(INT64 volumeCOM1, INT64 volumeCOM2, ThirdPartyDataIVAO* IVAOData, ThirdPartyDataVPILOT* VPILOTData) {
+    bool update = false;
+
+    if(_isInitialized == false) return;
+
+    if(IVAOData) {
```

Comment:
```suggestion
    if (IVAOData) {
```

You formatting seems to be a bit off.
Not going to mark all cases - maybe you check your formatter and maybe do format on safe)

Also I recommend 2 spaces instead of 4 for indentation for cpp